### ISSUE TITLE
exit("", x) string not updated

### ISSUE THREAD

**Issue by roland1977** [DIVGAMES/DIV-Games-Studio#18]
If you do the exit() statement into your program it returns the value and the correct string. Then, if you start the program again and quits with ALT+X it shows the latest return string.

Issues is: the string will not be updated!

Example:
program exit_test;

begin
	loop
		if (key(_esc)) exit("Exited with the ESC key1", 0); end
		frame;
	end
end

**Comment by CicTec** [comment]
Hello roland,

This problem only appears when running programs from the development environment, in the release version this does not happen.

Now the bug has been fixed, thanks for the report.

**Comment by MikeDX** [comment]
Cictec is there a pull request for this fix?

**Comment by CicTec** [comment]
Not yet Mike,

I'm running a re-format of all the core code for better bug fixes, then proceed to all bug fixes that are reported, once we finish, we'll get a pre-test build and when we confirm that all bugs are fixed well, we proceed with a single source code upload, instead of making multiple pull requests.

I think it is the best solution for now, but if you have a better idea, we can proceed in another way.

**Comment by MikeDX** [comment]
A single pull request with lots of changes will be difficult to track and check the differences of. In collaborative projects it is better to havesmaller pull requests that solve a single problem than one large upload which solves many (obviously sometimes one small change may fix more than one bug).

**Comment by CicTec** [comment]
Yes, but a re-format involve into all source code files of the core, are a lot of "changes" already.

I am recording in a worklog all the important changes and fixes where they were made, so that we can check it fast.
If this is not enough, i finish the refactor of the remaining files and proceed with the pull request of the one done so far.

**Comment by roland1977** [comment]
nice, thanks for fixing it :)
